src/main/java/io/jenkins/plugins/checks/github/status/GitHubSCMSourceStatusChecksTrait.java

```diff
@@ -136,6 +144,13 @@ protected void decorateContext(final SCMSourceContext<?, ?> context) {
      */
     @Extension
     public static class DescriptorImpl extends SCMSourceTraitDescriptor {
+        public static final boolean defaultSkipProgressUpdates;
+        public static final boolean enforceSkipProgressUpdates;
```

Comment:
these shouldn't be final / static, why do you need them in here? these will only get set on jenkins start